autoconnect/autoconnect-common/src/protocol.rs

```diff
@@ -66,6 +66,10 @@ pub enum ClientMessage {
         version: String,
     },
 
```

Comment:
Introducing new protocol messages should be kept behind a feature flag until we can properly verify that there are no problems with current or legacy UserAgents.